### ISSUE TITLE
Q-Drawer mini animate class is not removed after animation is finished, causing styling issues

### ISSUE THREAD

**Issue by PedroCavalheiro** [quasarframework/quasar#10093]
**Describe the bug**
When using Q-Drawer with mini, the class "q-drawer--mini-animate" isn't removed from the drawer container, causing styling issues when using any setting except bordered. This means, if using elevated, the icon will shift by one pixel. If using icons bigger than the default size, the problem is amplified and the icons get cut off.
**Codepen/jsFiddle/Codesandbox (required)**
https://jsfiddle.net/6aw7Ldtn/17/

**To Reproduce**
1. Go the jsfiddle above
2. Notice the icons are all fine. The first two icons are bigger, but they're still centered and displaying correctly.
3. Hover the drawer, notice all the icons have moved. The bigger ones are more obvious, but the small ones also move by one pixel since the drawer's style is elevated and not bordered.
4. Remove your mouse from the drawer. The big icons are completely wrong (off-center and cut-off), and the small ones are one pixel off-center.

**Expected behavior**
It is expected that the drawer returns to the initial state. It would also be desirable if the icons didn't move at all.

In other words, it is expected that the class "q-drawer--mini-animate" is removed from the div with the class "q-drawer-container" after the animation is done.

Consistency between elevated and bordered styles would also be expected. Bordered doesn't have thsi behavior by luck, since the border takes away one pixel from the drawer.

**Platform (please complete the following information):**
Quasar Version: v2.0.2
@quasar/app Version: v3.0.2

Quasar mode:
  - [X] SPA
  - [ ] SSR
  - [ ] PWA
  - [ ] Electron
  - [ ] Cordova
  - [ ] Capacitor
  - [ ] BEX

Tested on:
  - [x] SPA
  - [ ] SSR
  - [ ] PWA
  - [ ] Electron
  - [ ] Cordova
  - [ ] Capacitor
  - [ ] BEX

OS: Windows 10 Pro
Node: v14.17.3
NPM: 6.14.13
Yarn: Not applicable
Browsers: Google Chrome Version 91.0.4472.124, Firefox 89.0.2
iOS: Not applicable
Android: Not applicable (Drawer has different behavior on mobile)
Electron: Not applicable

**Comment by DG3D** [comment]
For those who got this issue and want a temp fix until quasar fix it
you can do this : 

```
<q-drawer v-model="drawer" show-if-above :mini="miniState" @mouseover="miniState = false" @mouseout="onMouseOut"`
```

```
  setup () {
    const miniState = ref(true)
    return {
      drawer: ref(false),
      miniState,
      menuList,
      onMouseOut (e) {
        miniState.value = true;
        const qdrawer = document.getElementsByClassName('q-drawer-container')[0]
        setTimeout(() => {
          qdrawer.classList.remove('q-drawer--mini-animate')
        }, 20)
      }
    }
  }
```

**Comment by rstoenescu** [comment]
Thanks for reporting.
Fix will be available in Quasar v2.2.1
